docs/references/sdk/types.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: SDK types
+description: "Learn about the three different types of SDKs: frontend-only, backend-only, and fullstack."
+---
+
+# SDK types
+
+Clerk categorizes its SDKs into three different types:
+
+- Frontend-only
+- Backend-only
+- Fullstack
+
+Take a look at your framework and check which type it enables. Then place your SDK into one of these three categories when first creating it. Arguably, creating a fullstack SDK is the most work so feel free to start off with frontend-only/backend-only SDK — you can still make it fullstack at a later point. Also, the framework you’re building the SDK for might evolve in the future, so maybe naturally your SDK will also need to change over time.
+
+| SDK Type | FAPI HTTP client | BAPI HTTP client | Hotloading ClerkJS | `window.Clerk` UI components | Authorization header verification | `__session` cookie verification | Handshake support |
+| - | - | - | - | - | - | - | - |
+| Frontend-only | ✅ | ⬜ | ✅ | ✅ | ⬜ | ⬜ | ⬜ |
+| Backend-only | ⬜ | ✅ | ⬜ | ⬜ | ✅ | ✅ | ✅ |
+| Fullstack | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
```

Comment:
🙃 This grey emoji, makes me think that something did not render correctly, may we switch to something else ?